src/main/java/Duke.java

```diff
@@ -1,10 +1,116 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final int MAX_TASK = 100;
+
+    static Task[] taskList = new Task[MAX_TASK];
```

Comment:
Perhaps you can use plural variable name for representing the collection of Task objects. Even though taskList is quite clear and can be easily understood, use of plural word like "tasks" could better convey the meaning.